rbac/management/group/view.py

```diff
@@ -206,6 +207,14 @@ def destroy(self, request, *args, **kwargs):
         @apiSuccessExample {json} Success-Response:
             HTTP/1.1 204 NO CONTENT
         """
+        group = get_object_or_404(Group, uuid=kwargs.get('uuid'))
+        if group.platform_default:
```

Comment:
Would we be able to throw a test in to validate this?

Reply:
Yeah, realized the need for that and working on it as we speak! Also adding the new field to the group serializer.

Reply:
Added tests in most recent commit